src/Components/SmartComponents/SystemsExposedTable/SystemsExposedTable.js

```diff
@@ -61,6 +64,8 @@ const SystemsExposedTable = ({ intl, cveName, cveStatusDetails, filterRuleValues
 
     const handleSelect = (payload) => dispatch(selectRows(payload));
 
+    useEffect(() => setIsAllExpanded(items.every(e => e.isOpen === true)), [items]);
```

Comment:
I think naming the current item for what it is is better for code readability, like here using `item` instead of `e` is more readable, it does not matter too much for one line lambda, but let's make that a standard practice